backend/src/main/java/feedzupzup/backend/notification/infrastructure/FcmPushNotifier.java

```diff
@@ -0,0 +1,93 @@
+package feedzupzup.backend.notification.infrastructure;
+
+import com.google.firebase.messaging.BatchResponse;
+import com.google.firebase.messaging.FirebaseMessaging;
+import com.google.firebase.messaging.MulticastMessage;
+import com.google.firebase.messaging.Notification;
+import feedzupzup.backend.notification.application.PushNotifier;
+import feedzupzup.backend.notification.domain.NotificationPayload;
+import feedzupzup.backend.notification.domain.NotificationToken;
+import feedzupzup.backend.notification.domain.NotificationTokenRepository;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class FcmPushNotifier implements PushNotifier {
+
+    private final NotificationTokenRepository notificationTokenRepository;
+    private final FirebaseMessaging firebaseMessaging;
+    private final FcmErrorHandler errorHandler;
```

Comment:
```suggestion
    private final FcmErrorHandler fcmErrorHandler;
```
p2)
멤버변수는 축약하지 않아야 가독성이 좋은 것 같아요!